.gitignore

```diff
@@ -32,3 +32,10 @@ playground.xcworkspace
 /.buckconfig.local
 /.buckd
 
+# CocoaPods
+*.xcworkspace
```

Comment:
Can we not ignore `AsyncDisplayKit.xcworkspace`? We've been using it to sync some project-wide settings across the team.

Also just realized that [this line](https://github.com/TextureGroup/Texture/blame/master/docs/_docs/development/how-to-develop.md#L12) in the dev doc is not correct. We should fix it as well!

Reply:
Sorry for not paying enough attention to it. Let's keep `.xcworkspace` files for this purpose. I've updated the commit to just ignore `Carthage` related files this time.

Reply:
Could you clarify why do we need to fix [this line](https://github.com/TextureGroup/Texture/blame/master/docs/_docs/development/how-to-develop.md#L12)? 

```
Run `pod install` in the directory that you cloned to.
```

Because the `Pods` folder is not kept under version control, it seems we need to run `pod install` after cloning the repo.

Reply:
ah, you're right. That line is fine.